<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="fname_layout">
      <item>
       <widget class="QLabel" name="fname">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Имя         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fname_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="lname_layout">
      <item>
       <widget class="QLabel" name="lname">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Фамилия </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lname_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="login_layout">
      <item>
       <widget class="QLabel" name="login">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Логин      </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="login_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="password_layout">
      <item>
       <widget class="QLabel" name="password">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Пароль   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_edit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="make_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>340</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
